fflib/src/classes/fflib_SObjectSelector.cls

```diff
@@ -33,7 +33,13 @@ public abstract with sharing class fflib_SObjectSelector
     /**
      * This overrides the Multi Currency handling, preventing it from injecting the CurrencyIsoCode fie ld for certain System objects that don't ever support it
      **/
-     private static Set<String> STANDARD_WITHOUT_CURRENCYISO = new Set<String> {'ApexClass', 'ApexTrigger', 'AsyncApexJob', 'Attachment', 'RecordType'};
+    private static Set<String> STANDARD_WITHOUT_CURRENCYISO = new Set<String> { AsyncApexJob.SObjectType.getDescribe().getName()
```

Comment:
This code change reverts https://github.com/financialforcedev/fflib-apex-common/issues/93 -- which was added to deal with a bizarre 'Entity is not api accessible' in PostInstall user context

Can you change these back to strings?